### PR TITLE
Create Features with Images Patterns

### LINKED ISSUE #110: Block Patterns - Features with Images
**Description**

🛠️ The goal of this issue is to track the implementation of the "Features with Images" pattern.
🎨 [Figma](https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=536%3A5856&mode=design&t=LHSPHKUrPq556l59-1)

Features with Images

### PR DESCRIPTION
**Pull request by huzaifaalmesbah**
Create Features with Images Patterns #110

### CODE REVIEW

**Comment by maneshtimilsina** [comment]
@huzaifaalmesbah You are using `get_stylesheet_directory_uri()` to load images from assets. It will not work if we use a child theme. It will be better to replace it with `get_template_directory_uri()` as discussed here https://github.com/WordPress/twentytwentyfour/issues/101

**Review by maneshtimilsina — CHANGES_REQUESTED** [review]
Please replace `get_stylesheet_directory_uri()` with `get_template_directory_uri()` to load images from assets so that the pattern will work with the child theme too.

**Comment by huzaifaalmesbah** [comment]
> Please replace `get_stylesheet_directory_uri()` with `get_template_directory_uri()` to load images from assets so that the pattern will work with the child theme too.

OK. thanks for the suggestion

**Comment by maneshtimilsina** [comment]
@huzaifaalmesbah I think `✳︎` needs to be translatable.

File: patterns/features-with-images.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ * Slug: twentytwentyfour/features-with-images
+ * Categories: gallery
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"6vh","bottom":"8vh"}}},"layout":{"type":"constrained","wideSize":"1280px"}} -->
+<div class="wp-block-group alignwide" style="padding-top:6vh;padding-bottom:8vh"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"align":"center","style":{"layout":{"selfStretch":"fit","flexSize":null},"spacing":{"padding":{"right":"0","left":"0"}},"elements":{"link":{"color":{"text":"var:preset|color|custom-borders"}}}},"textColor":"custom-borders","fontSize":"large"} -->
+<p class="has-text-align-center has-custom-borders-color has-text-color has-link-color has-large-font-size" style="padding-right:0;padding-left:0">✳︎</p>
```
Comment (maneshtimilsina) [inline patterns/features-with-images.php]: This icon/character `✳︎` needs to be translatable.

**Review by maneshtimilsina — APPROVED** [review]
Looks good to me.

File: patterns/features-with-images.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"4vh"} -->
+<div style="height:4vh" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|80"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"center"} -->
+<div class="wp-block-column is-vertically-aligned-center"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"align":"left","style":{"layout":{"selfStretch":"fit","flexSize":null},"spacing":{"padding":{"right":"0","left":"0"}},"elements":{"link":{"color":{"text":"var:preset|color|custom-borders"}}}},"textColor":"custom-borders","fontSize":"large"} -->
```
Comment (carolinan) [inline patterns/features-with-images.php]: Why does the padding need to be set to 0 here?

File: patterns/features-with-images.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<p class="has-text-align-left has-custom-borders-color has-text-color has-link-color has-large-font-size" style="padding-right:0;padding-left:0"><?php echo esc_html__( '✳︎', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"level":3,"fontSize":"large"} -->
+<h3 class="wp-block-heading has-large-font-size"><?php echo esc_html__( 'Études Architect App', 'twentytwentyfour' ); ?></h3>
```
Comment (carolinan) [inline patterns/features-with-images.php]: H3 uses the large font size by default, so this size should be removed from the pattern.

Comment (huzaifaalmesbah) [reply patterns/features-with-images.php]: Thanks for the review and your feedback. I will asap update PR.

**Comment by MaggieCabrera** [comment]
This is a duplicate of https://github.com/WordPress/twentytwentyfour/pull/69

**Comment by MaggieCabrera** [comment]
Closing in favor of #69

**Pull request closed without merging**